hail/Makefile

```diff
@@ -127,6 +127,7 @@ python/README.md: ../README.md
 $(PYTHON_JAR): shadowJar
 	cp -f $(SHADOW_JAR) $@
 
+PYTEST_ARGS = test/hail
```

Comment:
doesn't this overwrite whatever PYTEST_ARGS is?

I think we could do something like `PYTEST_TARGET ?= test/hail` and then use that above $(PYTEST_ARGS)

Reply:
This sets a default PYTEST_ARGS. If you pass PYTEST_ARGS to the make command, it will overwrite the default specified.

Reply:
don't you need `?=` for that?

Reply:
Also, this would break my setup, where I have `PYTEST_ARGS="--color=yes -vv"`. I'm happy to parameterize the target, we should leave extra args separate

Reply:
ah, would you like a TARGET = ?

Reply:
see my first comment, yeah

Reply:
> don't you need ?= for that?

You don't for the simple case (make pytest PYTEST_ARGS=" "). The variable specified on the command line will take precedence. It looks like ?= allows you to specify the variable **before** you invoke the make command, which is I think a win, so I'll change to ?=